recipes/example/meta.yaml

```diff
@@ -1,79 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please do not delete our example recipe. We need that for people to have something to base their recipes.

Reply:
reverted